lmdeploy/pytorch/backends/dlinfer/op_backend.py

```diff
@@ -40,6 +43,9 @@ def get_layer_impl_builder(cls, layer_type: OpType):
         elif layer_type == OpType.LinearW4A16:
             from .awq_modules import AwqLinearW4A16Builder
             return AwqLinearW4A16Builder
+        elif layer_type == OpType.RotaryEmbedding:
```

Comment:
What's the difference between DlinferApployRotaryEmbBuilder and DlinferRotaryEmbeddingBuilder?